package.json

```diff
@@ -69,9 +70,14 @@
     "react-native-tab-view": "^0.0.54"
   },
   "jest": {
-    "testPathDirs": [
```

Comment:
This config had issues with RN, see https://github.com/facebook/jest/issues/2670.